cron/config/config.yaml

```diff
@@ -21,3 +21,6 @@ bigquery-table: scorecard
 shard-size: 10
 webhook-url: 
 metric-exporter: stackdriver
+# UPGRADEv2: to remove.
+result-data-bucket-url2: gs://ossf-scorecard-data2
+bigquery-table2: scorecard2
```

Comment:
Unit tests for `config_test.go` might fail with this change if you don't update `config_test.go`

Reply:
updated. tests pass. I'll keep an eye on it.